integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -204,215 +227,42 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
    * under domain status.
    * Verifies that the new pod comes up and sample application deployment works.
    */
-  @Order(1)
   @Test
   @DisplayName("Test introSpectVersion starting a introspector and updating domain status")
   @Tag("gate")
   void testDomainIntrospectVersionNotRolling() {
+    // get the pod creation time stamps
+    LinkedHashMap<String, OffsetDateTime> pods = new LinkedHashMap<>();
```

Comment:
The type of the variable should just be `Map<String, OffsetDateTime>`; don't use the implementation class name here.

Reply:
Agreed, please fix this.

Reply:
Thanks for reviewing, fixed.